src/views/settings.hbs

```diff
@@ -1,4 +1,85 @@
-<div>
-    from Settings
-</div>
+<form class="settings">
+    <div class="user-info">
+        <div>
+            <label for="firstname">First Name :
+                <input id="firstname" type="text" value="{{userInfo.firstname}}" required>
+            </label>
+        </div>
+        <div>
+            <label for="lastname">Last Name :
+                <input id="lastname" type="text" name='lastname' value="{{userInfo.lastname}}" required>
+            </label>
+        </div>
+    </div>
+
+    <div>
+        <div>
+            <label> City : </label>
+            <select id="city-fields">
+                <option value="1"> Khanyounis </option>
+                <option value="2"> Gaza </option>
+                <option value="3"> Rafah </option>
+                <option value="4"> Dier-Al balah </option>
+            </select>
+        </div>
+        <div>
+            <label for="birthday">Birthday Date :
+                <input id="birthday" type="text" name='birthday' placeholder="__/__/____" required>
```

Comment:
@Angham116 This could be an `<input type="date"` if you want, that might be easier for people to use (though that might take some extra work to parse and put into your database birthday field..).  Also, you could set the value to `{{userInfo.birthday}}`.